LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentBrowserSwitchRequest.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+
+/**
+ * Request parameters associated with a [LocalPaymentPendingRequest.Started]
+ */
+class LocalPaymentBrowserSwitchRequest internal constructor(
+    browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
+) {
+
+    private var browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
+
+    init {
+        this.browserSwitchPendingRequest = browserSwitchPendingRequest
+    }
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Is this annotation needed in Kotlin?

Could we make `LocalPaymentBrowserSwitchRequest` internal which would obfuscate the entire class from the merchant?

Reply:
See comment above - will fix this across modules in a separate PR 👍